QuickOSM/core/process.py

```diff
@@ -239,7 +239,7 @@ def process_quick_query(
         osm_objects=osm_objects,
         timeout=timeout
     )
-    query = query_factory.make(QueryLanguage.Oql)
+    query = query_factory.make(QueryLanguage.OQL)
```

Comment:
We should have some Pycharm warning for such error, no ?

Reply:
No it remained silent